py3status/modules/battery_level.py

```diff
@@ -302,6 +305,67 @@ def _parse_battery_info(sys_path):
             battery_list.append(battery)
         return battery_list
 
+    def _extract_battery_information_from_upower(self):
+        """
+        Get the output with command 'upower --dump'
+        An example of old laptop with dead battery.
+
+        Device: /org/freedesktop/UPower/devices/battery_BAT0
+          native-path:          BAT0
+          model:                DELL RN8878
+          serial:               17000
+          power supply:         yes
+          updated:              Tue Sep 19 06:46:31 2017 (67 seconds ago)
+          has history:          yes
+          has statistics:       yes
+          battery
+            present:             yes
+            rechargeable:        yes
+            state:               charging
```

Comment:
I need to know what happen when we discharge here. Does it say `discharging`? In that case, we might want to tweak `'charging': 'charging' in output['state'],` boolean below. That's the only issue, I think.